easybuild/easyconfigs/a/Autotools/Autotools-20170619-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::     Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::     GNU GPL-v3.0
```

Comment:
if i recall right, EB under GPLv2 so this is potentially a redistribution issue...

Reply:
Yes, indeed, we ship `easybuild-easyconfigs` packages under GPLv2.

On the other hand, there's very little creativity involved with composing easyconfig files, so I'm not sure whether this is really an issue...

@robqiao Can you please reconsider either changing the license to something compatible with GPLv2, or simply dropping the license line?

Reply:
To clarify: the license you specify here is assumed to be the license for the easyconfig file, not the license for the software it is installing...